build.cmd

```diff
@@ -4,51 +4,96 @@ PUSHD %~dp0
 SETLOCAL
 SETLOCAL ENABLEDELAYEDEXPANSION
 
+:Loop
+if [%1]==[] GOTO Begin
+
+if /I [%1]==[Debug] (
+    SET Configuration=%1
+    GOTO Next
+)
+
+if /I [%1]==[Release] (
+    SET Configuration=%1
+    GOTO Next
+)
+
+if /I [%1]==[PROD] (
+    SET Environment=PROD
+    GOTO Next
+)
+
+if /I [%1]==[raw] (
+    SET UpdateTemplate=%1
```

Comment:
greate advice!

---
In reply to: [68715890](https://github.com/dotnet/docfx/pull/435#discussion_r68715890) [](ancestors = 68715890,68714739,68713739)